vertx-web/src/main/java/io/vertx/ext/web/handler/impl/CorsHandlerImpl.java

```diff
@@ -101,10 +102,6 @@ public CorsHandler exposedHeaders(Set<String> headerNames) {
 
   @Override
   public CorsHandler allowCredentials(boolean allow) {
-    if (allowedOrigin == null && allow) {
```

Comment:
Why removing this?

Reply:
As far as I understood from [MDN](https://developer.mozilla.org/en-US/docs/Web/HTTP/Server-Side_Access_Control#Simple_cross-site_requests), you can do a configuration with wildcard as allowed origin and allow credentials = true, but the server must echo back the origin